UI/CUIControls.h

```diff
@@ -608,23 +608,36 @@ class ProductionInfoPanel : public CUIWnd {
 /** Displays progress that is divided over mulitple turns, as in the Research and Production screens. */
 class MultiTurnProgressBar : public GG::Control {
 public:
-    /** \name Structors */ //@{
-    MultiTurnProgressBar(int total_turns, double turns_completed, double total_cost, double turn_spending,
-                         const GG::Clr& bar_color, const GG::Clr& background, const GG::Clr& outline_color);
-    //@}
-
-    /** \name Mutators */ //@{
+    /** @name Structors */ //@{
+    /** Ctor
+    * @param[in] num_segments Number of segments in the bar
+    * @param[in] percent_completed Percent of bar to fill
+    * @param[in] percent_predicted Percent of bar to fill after
```

Comment:
It's not clear what it the valid input range for this.

 * 0.0 to (1.0 - percent_completed)?
 * percent_completed to 1.0?
 * 0.0 to 1.0?